src/main/resources/features/theme 1/07 - tester records contingency.feature

```diff
@@ -160,8 +160,7 @@ Feature: 07 - Tester records Contingency Test
     And I check the vehicle summary section of the test summary has "Registration number" of {registration1}
     And I check the vehicle summary section of the test summary has "VIN/Chassis number" of {vin1}
     And I check the brake results section of the test summary is "None Recorded"
-    And I check the dangerous failures section of the test summary has "None recorded"
```

Comment:
I think this is required now we are over 4 days past EURW switchover

Reply:
Updated